src/PaymentServer/Main.hs

```diff
@@ -90,6 +93,7 @@ data ServerConfig = ServerConfig
   , database        :: Database
   , databasePath    :: Maybe Text
   , endpoint        :: Endpoint
+  , stripeKey    :: ByteString
```

Comment:
I realized that I made a mistake having `signing-key` as an option.  Instead it should be `signing-key-path`, I think, so that the key itself never needs to make it into `argv`.  The same goes for `stripeKey`, I think.  But since this has to be fixed for signing key too I think it makes sense to leave it as-is and then do a follow up where they both get turned into path arguments instead of literal key arguments.

Reply:
https://github.com/PrivateStorageio/PaymentServer/issues/31

Reply:
Thanks for the suggestion. I will look at that next.

Reply:
I also fixed the formatting.